bip-musig2.mediawiki

```diff
@@ -514,6 +524,10 @@ The reference implementation is for demonstration purposes only and not to be us
 === Modifications to Nonce Generation ===
 
 Implementers must avoid modifying the ''NonceGen'' algorithm without being fully aware of the implications.
+
+One potential pitfall emerges when ''NonceGen'' is altered such that ''k<sub>1</sub>'' and ''k<sub>2</sub>'' are drawn uniformly at random instead of setting them to the output of a ''hash'' function that includes ''pk'' as input.
+The result is that in some situations, an adversary can forge a signature by influencing the individual tweak of a victim signer after seeing the ''pubnonce'' (see [https://github.com/jonasnick/musig2-tweaking here] for the details).
```

Comment:
You're right, that's not really clear after the "add public key to secnonce" commit. Should mention that this concern only applies when this is not done (which is what I expect to happen in practice because implementers probably have better data structures for this).

Reply:
Ok I fixed the paragraph.